[new] add pqGrid@2.0.4 with git auto-update

Amomo opened the pull request:
Hi @maruilian11  

For #6298, I add pqGrid at v2.0.4 and add git auto-update config in its package.json.
repo: https://github.com/paramquery/grid

Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `56`, Star `316`, Fork `142`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

PeterDaveHello commented on ajax/libs/pqGrid/package.json:
```diff
@@ -0,0 +1,48 @@
+    "email": "pdhindsa@paramquery.com",
+    "url": "http://www.paramquery.com/"
+  },
+  "licenses": "GPL-3.0",
```
licenses?

Amomo replied:
I have fix it, thank you! :-)

PeterDaveHello commented on ajax/libs/pqGrid/package.json:
```diff
@@ -0,0 +1,48 @@
+    "files": [
+      "pqgrid*.+(js|css)",
+      "images/*.+(png|gif)"
+    ]
```
What about the localization files?

Amomo replied:
thanks for reminding me that, I almost forget!
They are added now. :-D

PeterDaveHello replied:
but without minified version ...

maruilian11 commented:
@Amomo could you please tell me why we needs files under `localize`. thanks.

Amomo commented:
Because this lib provide a UI, with these `localize` files, the lib would be international for globe user.

Amomo commented:
@maruilian11 I added the minified `localize/` files to "requiredFiles" field.
And, I have rebased it on latest master branch.
Would you mind to check it again? Thank you. :-D

maruilian11 commented:
@Amomo i think this PR is ok. could you rebase it again?

Amomo commented:
@maruilian11 I have rebase this PR on latest mater branch.
Could you help me check it again? Thank you~ :-D

maruilian11 commented:
@Amomo sorry... i'm late..😭

Amomo commented:
@maruilian11 it's okay~ Could you help me check it again? Thank you~ :-D

maruilian11 commented:
@PeterDaveHello i think the PR is ok.

PeterDaveHello commented:
No theme? https://github.com/paramquery/grid/tree/master/themes/Office

Amomo commented:
@maruilian11 I have added the theme of this lib, and rebased this PR on latest master branch.
Could you help me checking this PR again? Thank you! :-)

maruilian11 commented:
@PeterDaveHello i think this PR is ok.

PeterDaveHello commented:
For the complex filemap, maybe you can ask author if she/he would like to create a dist folder for distribution files!
